test/inspector/test-on-off.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+const common = require('../common');
+common.skipIfInspectorDisabled();
+const helper = require('./inspector-helper.js');
+
+function testResume(session) {
+  session.sendCommandsAndExpectClose([
+    { 'method': 'Runtime.runIfWaitingForDebugger' }
+  ]).expectStderrOutput('Debugger listening on');
+}
+
+function runTests(harness) {
+  harness
+    .runFrontendSession([
+      testResume,
+    ]).expectShutDown(0);
+}
+
+const script = 'process._debugEnd();' +
```

Comment:
It is not a new functionality. Both APIs were already there (_debugProcess is meant to attach to other processes - but looks like it can be made to work on self just fine).

_debugProcess sends a signal on Posix systems. It is the function used to attach to processes that were not started with --inspect.
__debugEnd stops the agent. Looks like it had been there for ages, personally I am not sure why it is needed :)